.github/workflows/tf_apply_production.yml

```diff
@@ -52,3 +52,7 @@ jobs:
       - name: Apply api
         working-directory: terragrunt/env/production/api
         run: terragrunt apply --terragrunt-non-interactive -auto-approve
+
```

Comment:
Could you move this above the `Apply api` step.  Once we've got this merged, I'll help you update the `api` Terragrunt configuration to show the module dependency ordering.

Reply:
Done!